### DISCUSSION TITLE
ESP32-audioI2S audio.connecttoSD failed

### DISCUSSION THREAD

**Discussion by pclai930** [schreibfaul1/ESP32-audioI2S#344] [General]
I have a project under PlateformIO with espressif32 platform and arduino framework with an ESP32-A1S audio devkit board.
I use the library schreibfaul1/ESP32-audioI2S.
The platform espressif 32 has recently been upgraded to version 5.0.
I have a problem with the instruction "audio.connecttoSD()" which can't read the audio files on the SD card anymore.
Everything was working fine before the upgrade to espressif32 version 5.0.
I would like to know if anyone has encountered this problem.

**Comment by daywink** [comment]
Try using the SdFat library, it works for me.
